src/azure-cli/azure/cli/command_modules/monitor/aaz/latest/monitor/log_analytics/workspace/table/_update.py

```diff
@@ -101,7 +101,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
             arg_group="Properties",
             help="The table total retention in days, between 4 and 2555.",
             fmt=AAZIntArgFormat(
-                maximum=2555,
+                maximum=2556,
                 minimum=4,
```

Comment:
Auto generated code needs to be updated in AAZ data model.

Reply:
Please refer to https://github.com/Azure/azure-rest-api-specs/blob/main/specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2021-12-01-preview/Tables.json#L656

Reply:
@AllyW is this a task you are asking of me? I don't understand what the next step is to getting my fix merged.

Reply:
Hi @ryan-royals , thanks for code testing and fixing. The thing is, cli code is auto-generated by azure service docs posted upside, so this problem should be solved by service doc update and I'm following it. Therefore, code change here is not enough and I'll keep you updated if anything changes.

Reply:
Hi @AllyW, any updates on this?

Reply:
hi @ryan-royals, service does not schedule this doc change recently and we are planning to apply your changes in this sprint. As is known to you that, _update.py is generated code by doc. If we need to adjust the `min` and `max` validators, the correct way is inheriting from this generated cmd and editing its fmt in new class. I'm going to do this in this sprint and link the pr here. Also if you're interested, you can refer to this pr https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/network/operations/watcher.py#L333